test-suite/js/scalajs-test-suite-2.10.html

```diff
@@ -3,15 +3,24 @@
 <head>
   <title>Scala.js Test Suite - Jasmine HTML reporter</title>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/jasmine/1.3.1/jasmine.css"></link>
 </head>
 <body>
 <script type="text/javascript" src="http://cdn.jsdelivr.net/jasmine/1.3.1/jasmine.js"></script>
```

Comment:
Note that this is kept because there are still a couple of tests that use info from the jasmine framework. This will be removed in the next PR.